ui/page/main_page.go

```diff
@@ -664,3 +660,65 @@ func (mp *MainPage) LayoutTopBar(gtx layout.Context) layout.Dimensions {
 		}),
 	)
 }
+
+func (mp *MainPage) desktopNotifier(notifier interface{}) {
```

Comment:
Isn't it better to have this page create its own notification listeners and do as it pleases rather than rely on sub pages calling this method when necessary? Also makes the code clearer that way.

Reply:
Those are not listeners, It's a func used to post desktop notification from the main page. I have added comment to the code to clarify that section of the code.